internal/xds/translator/ratelimit.go

```diff
@@ -513,54 +504,48 @@ func descriptorsEqual(a, b *rlsconfv3.RateLimitDescriptor) bool {
 //   - ...
 func addRateLimitDescriptor(
```

Comment:
Nit: can we rename `addRateLimitDescriptor` to a more descriptive name? The current name might be a bit hard to follow for later readers if they are't  very fimilar with the server side rate limit configuration.

Maybe we could have a shared `addRateLimitDescriptor` helper, and split out `addRouteMatchToDescriptors` and `addPolicyMatchToDescriptors` for clarity?